test/sequential/test-watch-mode.mjs

```diff
@@ -574,4 +576,83 @@ console.log(values.random);
       `Completed running ${inspect(file)}`,
     ]);
   });
```

Comment:
Maybe you meant to add it here?
```suggestion
  });

```

Reply:
Yeah, I think this might have happened while resolving merge conflicts